src/tests/ftest/security/cont_owner.yaml

```diff
@@ -0,0 +1,22 @@
+hosts:
+  test_servers: 1
+  test_clients: 1
+timeout: 1200
```

Comment:
The test took at most 2 minutes per variant here:
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14290/14/testReport/FTEST_security/DaosContainerOwnerTest/

Maybe reduce this timeout to 180? (3 minutes)

Reply:
I just copied this section from the existing cont_acl.yaml to have something here. I can adjust this.

Reply:
Fixed